test/LockedAccount.js

```diff
@@ -1029,18 +1038,65 @@ contract(
           "setPenaltyDisbursal"
         ];
         PublicFunctionsAdminOnly.forEach(name => {
-          it(`${name}`);
+          it(`${name}`, async() => {
+            let pendingTx;
+            migrationTarget = await deployMigrationTarget(
+              assetToken,
+              operatorWallet
+            );
+            switch(name) {
+              case "enableMigration":
+                await migrationTarget.setMigrationSource(lockedAccount.address, {
+                  from: admin
+                });
+                pendingTx = lockedAccount.enableMigration(migrationTarget.address, {
+                  from: investor
+                });
+                break;
+              case "setController":
+                pendingTx = lockedAccount.setController(admin, { from: investor });
+                break;
+              case "setPenaltyDisbursal":
+                pendingTx = lockedAccount.setPenaltyDisbursal(testDisbursal.address, {
+                  from: investor
+                });
```

Comment:
i would throw here on default to indicate programmer error — to avoid typos in strings or whatever

Reply:
done - linter told me